rllib/utils/test_utils.py

```diff
@@ -51,11 +51,11 @@ def framework_iterator(config=None,
             logger.warning("framework_iterator skipping {} (tf not "
                            "installed)!".format(fw))
             continue
-        elif fw == "eager" and not eager_mode:
+        elif fw == "tfe" and not eager_mode:
```

Comment:
Could we split this PR into a couple pieces:
1) add the basic infrastructure code and a simple example
2) port rest of examples

Reply:
Was not supposed to be here. Reverted this.

Reply:
Done.